src/naive/datetime/mod.rs

```diff
@@ -606,6 +634,34 @@ impl NaiveDateTime {
         Some(NaiveDateTime { date, time })
     }
 
+    /// Subtracts given `Months` to the current date and time.
```

Comment:
Sorry, minor nit I forgot to mention: "Subtracts .. to" should use have "from" instead of "to".